mkosi/resources/man/mkosi.1.md

```diff
@@ -204,80 +204,67 @@ The following command line verbs are known:
 
 ## Commandline-only Options
 
-Those settings cannot be configured in the configuration files.
+The following settings cannot be configured in the configuration files:
+
+`--help`, `-h`
+:   Show brief usage information.
 
 `--force`, `-f`
-:   Replace the output file if it already exists, when building an
-    image. By default when building an image and an output artifact
-    already exists `mkosi` will refuse operation. Specify this option
-    once to delete all build artifacts from a previous run before
-    re-building the image. If incremental builds are enabled,
-    specifying this option twice will ensure the intermediary
-    cache files are removed, too, before the re-build is initiated. If a
-    package cache is used (also see the **Files** section below),
-    specifying this option thrice will ensure the package cache is
-    removed too, before the re-build is initiated. For the `clean`
-    operation this option has a slightly different effect: by default
-    the verb will only remove build artifacts from a previous run, when
-    specified once the incremental cache files are deleted too, and when
-    specified twice the package cache is also removed.
+:   Replace the output file if it already exists, by default `mkosi`refuses to overwrite an existing image.
+    This option can be specified once, twice or trice to change the scope of artifact removals.
+    Specify once to delete all build artifacts from a previous run. Twice to include removal of the
+    intermediary package cache from incremental builds. Thrice to remove all build artifacts and the the full
+    package cache. See the **Files** section for more context and details.
+    If `mkosi` is followed by the `clean` verb, the `-f` option can only be specified twice, see the
+    description of the command line verbs above.
 
 `--directory=`, `-C`
-:   Takes a path to a directory. `mkosi` switches to this directory before
-    doing anything. Note that the various configuration files are searched
-    for in this directory, hence using this option is an effective way to
-    build a project located in a specific directory.
+:   Specify a path to a directory. `mkosi` switches to this directory before doing anything. Note that the
+    various configuration files are searched for in this directory by default. Using this option can be used
+    to take advantage of defaults when executing mkosi inside of a profile directory or to execute `mkosi` in
+    any other directory deemed useful.
```

Comment:
I don't think this improves on the previous version. "Takes a path" is a common way of putting this, since functions take arguments and in a more abstract sense mkosi is a function and this is a way to pass such an argument. 

The "by default" at the end of third sentence is superfluous with respect to the rest of the sentence and the last sentence is a lot of words that say very little. It should also not mention profiles, since that's a different concept in mkosi that doesn't have anything to do with this.

Reply:
My goal is to start each description with the value and default for the option or setting at hand.  Currently this is inconsistent, hampering the ability to skim over the document. This is another "general principle" I'd like to defend.

Reply:
For less technical users, and by including "value", it's pretty obvious we are talking about what the option "takes", "accepts", "honors" or anything else.